addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -166,12 +169,25 @@ public void handleCommand(final ChannelUID channelUID, final Command command) {
             case ChromecastBindingConstants.CHANNEL_VOLUME:
                 handleVolume(command);
                 break;
+            case ChromecastBindingConstants.CHANNEL_PLAY_URI:
+                handlePlayUri(command);
+                break;
             default:
                 logger.debug("Received command {} for unknown channel: {}", command, channelUID);
                 break;
         }
     }
 
+    private void handlePlayUri(Command command) {
+        if (command instanceof StringType) {
+            if (command.equals("NULL")) {
```

Comment:
This evaluates to true if you send NULL to the REST API. However if the .isEmpty() is semantically the same I'll update the PR.